src/reducers/viewconfig.ts

```diff
@@ -165,7 +174,7 @@ const node = (state: NormalizedComponent | null, action: Action, items: Normaliz
                 name: createName(action.payload.component.type, id)
             };
         }
-        case 'MODIFY_VIEW_CONFIG_NODE':
+        case 'MODIFY_VIEW_CONFIG_NODE':  // TODO: Make sure it get's normalized in case of total component switch. It doesn't break, but it's not nice either
             return {
                 ...action.payload.component,
                 id: state!.id,
```

Comment:
id & childIds zitten al in de payload, van waar het overschrijven met de state values?